api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -135,12 +166,12 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	public Drug getDrug(Integer drugId) throws DAOException;
 	
 	/**
-	 * DAO for retrieving a list of drugs based on the following criteria
+	 * Retrieve a list of drugs based on the supplied criteria
 	 * 
-	 * @param drugName
-	 * @param concept
-	 * @param includeRetired
-	 * @return List&lt;Drug&gt;
+	 * @param drugName The name of the drug to required
```

Comment:
```suggestion
	 * @param drugName The name of the drug to search for
```

Reply:
oops thanks for this , i missed this out .